packages/stan/README.md

```diff
@@ -0,0 +1,203 @@
+console.log( registry.get( counter ) ); //  prints 1.
+
+// Modify the value of the counter
+registry.set( counter, 10 );
+
+console.log( registry.get( counter ) ); //  prints 10.
+```
+
+### Subscribing to changes
+
+Each atom is an observable to which we can subscribe:
+
+```js
+registry.subscribe( counter, () => {
+    console.log( registry.get( counter ) );
+} );
+
+registry.set( counter, 2 ); // prints 2.
+registry.set( counter, 4 ); // prints 4.
+```
+
+### Derived atoms
+
+Atoms can also derive their value based on other atoms. We call these "derived atoms".
+
+```js
+import { createAtom, createDerivedAtom } from '@wordpress/stan';
+
+const counter1 = createAtom( 1 );
+const counter2 = createAtom( 2 );
+const sum = createDerivedAtom(
+    ( { get } ) => get( counter1 ) + get( counter2 )
+);
+```
+
+In the example above, we create two simple counter atoms and third derived "sum" atom which value is the sum of both counters.
+
+```js
+console.log( registry.get( sum ) ); // prints 3.
+
+// Adding a listener automatically triggers the refreshing of the value.
+// If the atom has no subscriber, it will only attempt a resolution when initially read. 
+// But it won't bother refreshing its value, if any of its dependencies change.
+// This property (laziness) is important for performance reasons.
+sumInstance.subscribe( () => {
+    console.log( registry.get( sum ) );
+} );
+
+// This edits counter1, triggering a resolution of sumInstance which triggers the console.log above.
+registry.set( counter1, 2 ); // now both counters equal 2 which means sum will print 4.
+registry.set( counter1, 4 ); // prints 6
+```
+
+### Async derived atoms
+
+Derived atoms can use async functions to compute their values. They can for instance trigger some REST API call and returns a promise.
+
+```js
+const sum2 = createDerivedAtom(
+    async ( { get } ) => {
+        const val1 = await Promise.resolve(10);
+        return val1 * get( counter );
+    } 
+);
+```
+
+The value of async atoms will be equal to `null` until the resolution function finishes.
+
+### Bindings
+
+It is important to note that stan instance and registries API are low-level APIs meant to be used by developpers to build bindings for their preferred frameworks. By in general, a higher-level API is preferred.
+
+Currently available bindings:
+
+-   `@wordpress/data`: WordPress data users can continue to use their existing high-level APIs useSelect/useDispatch (selectors and actions) to access the atoms. The selectors are just high-level atoms that can rely on lower-level ones and the actions are just functions that trigger atom setters. The API for `@wordpress/data` store authors to bridge the gap is still experimental.
+
+## API Reference
+
+<!-- START TOKEN(Autogenerated API docs) -->
+
+<a name="createAtom" href="#createAtom">#</a> **createAtom**
+
+Creates a basic atom.
+
+_Parameters_
+
+-   _initialValue_ `T`: Initial Value in the atom. \*
+-   _config_ (unknown type): Common Atom config.
```

Comment:
This bug with autogenerated API is really annoying, let's tackle it in a follow-up.